src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -303,13 +318,15 @@ def query_widgets
           ]
         end
 
+        # @macro seeAbstractWidget
         def store
           start_block, end_block = query_widgets
           len = end_block - start_block + 1
           bsize = @regions.first.block_size # where does this come from?
           @ptemplate.region = Y2Storage::Region.create(start_block, len, bsize)
         end
 
+        # @macro seeAbstractWidget
```

Comment:
@mvidner well, 90% is good if yard see ancestors, as by default it use documentation from ancestor for it. But in this case it do not see it. So I am fine if we simply descrease documentation coverage or remove it at all ( and keep only checking that yard is valid ) with big TODO.